lib/AST/RequirementMachine/RequirementLowering.cpp

```diff
@@ -759,6 +759,20 @@ void swift::rewriting::realizeInheritedRequirements(
 
     realizeTypeRequirement(dc, type, inheritedType, loc, result, errors);
   }
+
+  // All associated types or generic type params are Copyable by default.
+  // TODO: if we saw ~Copyable in the inherited types list earlier, skip this!
+  if (auto *copyable = ctx.getProtocol(KnownProtocolKind::Copyable)) {
+    SourceLoc loc = decl->getLoc();
+    Type copyableType = copyable->getDeclaredInterfaceType();
+
+    // Add Copyable as an "inferred" requirement.
+    SmallVector<Requirement, 2> reqs;
+    Requirement sugaredReq(RequirementKind::Conformance, type, copyableType);
+    desugarRequirement(sugaredReq, loc, reqs, errors);
```

Comment:
desugarRequirement() is going to be a no-op in this case. you can just add the new requirement to `result`

Reply:
thank you!